packages/playwright-core/types/types.d.ts

```diff
@@ -14616,6 +14649,14 @@ export interface Response {
    */
   headerValues(name: string): Promise<Array<string>>;
 
+  /**
+   * > NOTE: This field is Chromium only. It's safe to call when using other browsers, but it will always be `null`.
+   *
+   * If set, indicates whether this Response was fullfilled by a Service Worker's Fetch Handler (i.e. via
+   * [FetchEvent.respondWith](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/respondWith)).
+   */
+  isFromServiceWorker(): null|boolean;
```

Comment:
Why `null` here? I'd return `false` for other browsers.

Reply:
Unresolving—we're going to see if we can get the correct information here across all the browsers.

Reply:
Per-offline comments, we'll land as it and follow up.